controllers/metrics/runnerdeployment.go

```diff
@@ -0,0 +1,37 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/summerwind/actions-runner-controller/api/v1alpha1"
+)
+
+const (
+	rdName      = "rd_name"
+	rdNamespace = "rd_namespace"
```

Comment:
Perhaps it would be a bit better to use names that are similar to those of kube-state-metrics? [In kube-state-metrics, the namespace label is just `namespace` regardless of the kind and the name label is the lowercase kind name](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/deployment-metrics.md):

```suggestion
	rdName      = "runnerdeployment"
	rdNamespace = "namespace"
```

Reply:
Good point 👍 
I changed the labels to kube-state-metrics style. It would be better for consistency between various components such as kube-state-metrics.